inc/container.class.php

```diff
@@ -277,6 +277,21 @@ static function getSpecificValueToDisplay($field, $values, array $options = [])
             $i     = 1;
             foreach ($types as $type) {
                $name_type = getItemForItemtype($type);
+               
+               // Handle case when item type doesn't exist.
+               // Function getItemForItemtype() returns false on error, instead of an object.
+               // Use case: When deleting a GenericObject that has Fields.
+               if( ! is_object($name_type)) {
+                  throw new Exception(
+                     sprintf(
+                        'Undefined item type "%s" in column "%s" of database table "%s".%s',
+                        $type,
+                        'itemtypes',
+                        'glpi_plugin_fields_containers',
+                        ' Check for recently deleted custom objects that had custom fields.'
+                  ));
```

Comment:
Exceptions are not correctly thrown in GLPI, trigerring a warning would be a better solution.
```suggestion
                  trigger_error(
                     sprintf(
                        'Unknown item type "%s" in column "%s" of database table "%s".%s',
                        $type,
                        'itemtypes',
                        'glpi_plugin_fields_containers',
                        ' Check for recently deleted custom objects that had custom fields.'
                     ),
                     E_USER_WARNING,
                  );
```